vagrant/Vagrantfile

```diff
@@ -59,10 +65,16 @@ Vagrant.configure("2") do |config|
   #   # Display the VirtualBox GUI when booting the machine
   #   vb.gui = true
   #
-  #   # Customize the amount of memory on the VM:
-    vb.memory = "4096"
+    vb.memory = VM_MEMORY
+    vb.cpus = VM_CPUS
   end
-  #
+
+  config.vm.provider :libvirt do |lv|
```

Comment:
Does adding this libvirt provider support change the way users would bring up the vagrant box or ssh into the VM outlined [here](https://github.com/tern-tools/tern#getting-started-with-vagrant)? If so, we'll need to update the README.

Reply:
As for vagrant up, vagrant ssh, vagrant destroy, etc - works the same as with virtualbox provider (or any other provider for that matter).
https://github.com/vagrant-libvirt/vagrant-libvirt#features
As for `Install` section I think its best to leave it as is- Vagrant uses Virtualbox by default, I think its best not to confuse first-time users with too much options, while people who use different providers will be able to add support for their choice and possibly PR to upstream. Let me know if you think different, I'll add a commit with README Install step addition.

Reply:
Ok, as long as users don't need to install additional libvirt dependencies to make vagrant work we can leave the README as is.

Reply:
vagrant will figure out which provider to use, as far as I remember, unless specified with --provider or ENV var, Vagrant will use first usable provider it finds supported in the Vagrantfile (and it goes in order of definition so virtualbox is first, libvirt is second)